### DISCUSSION TITLE
Feature Request: Make is possible to distinguish between user-triggered & API-trigger pause events

### DISCUSSION THREAD

**Discussion by George-Evans** [vimeo/player.js#676] [Ideas]
We have a use case where we need to trigger different behaviour depending on whether a video is paused by the user, or by the API/autopause etc. 

The specific example is a page with several videos, that each play when they enter the viewport, while all the others are autopaused. However, we'd like to implement the behaviour that if a user manually pauses a video, it will stay paused and not play again if the user scrolls back to it. 

As far as I can see, there is just the one `pause` event, which is fired by user click, autopause or by `player.pause();`.

**Comment by fisherinnovation** [comment]
Thanks for the feature request submission! This is also something we have discussed internally as well. Leaving this ticket open to track any updates made around this idea.

**Comment by George-Evans** [comment]
Thanks! Glad to hear it's something that's already been considered! I imagine there would be cases where doing the same for the play event may also be useful. Interested to see what comes out if it 😊

**Comment by Rikcon** [comment]
Hello !
Would be great to get some feedback on status of this issue, i need it for my project asap.

Use case:
So, on pause i need to open a modal window asking if student is ready to complete the task.
Also i have auto next video switching if there's no task for this video, that's when weird things start happening, vimeo player sends pause event with video switching, then loaded, and then cuechange.
So my code thinks that a person clicked pause, and guess what happens ? Yes, a unwanted modal window appears frustrating my students. 
I though i could fix that just by tracking seconds played, but it appears to be somehow cached or bugged, sometimes i get seconds from previous video ( e.g i watched 20 sec. of video, switched video programmatically and i get paused event with obsolete seconds value), i very much rely on seconds in my code because of reasons i explained above. 

I'm attaching a screenshot of my console when this bug happens, i understand that this issue is not the place to discuss this bug, i'm only saying that solving this issue, or issue with irrelevant seconds will fit my needs perfectly.

Thank you in adavance !

![image_console](https://user-images.githubusercontent.com/1577550/63156560-99e4bd00-c01d-11e9-9e44-b24c105fabc3.png)
